examples/vuejs/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["Vue.volar", "Vue.vscode-typescript-vue-plugin"]
```

Comment:
Do we really have to store an editor-dependent file under Git ?

Reply:
It's default installation of default Vue app. Didn't touch it.